jpos/src/main/java/org/jpos/q2/Q2.java

```diff
@@ -83,7 +83,7 @@
  * @author <a href="mailto:vsalaman@vmantek.com">Victor Salaman</a>
  */
 @SuppressWarnings("unchecked")
-public class Q2 implements FileFilter, Runnable {
+public class Q2 implements DirectoryStream.Filter<Path>, Runnable {
```

Comment:
Is it possible to leave the FileFilter interface? I don't think it's a big deal but might help minimize impact for anyone updating to this version